src/scanpy/_utils/compute/is_constant.py

```diff
@@ -81,7 +81,7 @@ def is_constant(
 def _(a: NDArray, axis: Literal[0, 1] | None = None) -> bool | NDArray[np.bool_]:
     # Should eventually support nd, not now.
     if axis is None:
-        return (a == a.flat[0]).all()
+        return bool((a == a.flat[0]).all())
```

Comment:
This makes the function return a `bool` instead of a `np.bool_` scalar as the type hints say.

Behaves almost the same but has a different repr so passes the doctests.